cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -179,6 +179,32 @@ def _to_timestamp(json_string):
 )
 
 
+class FakeEngineClient(engine_client.EngineClient):
+    """Fake engine client for testing."""
+
+    def __init__(self):
+        super().__init__()
+        self._processor = quantum.QuantumProcessor()
+
+    async def get_processor_async(
+        self, project_id: str = "", processor_id: str = ""
+    ) -> quantum.QuantumProcessor:
+        return self._processor
+
+    get_processor = duet.sync(get_processor_async)
+
+    def set_processor(self, processor: quantum.QuantumProcessor):
```

Comment:
It looks like we call this just after construction and never again. Could the value just be passed to the constructor instead?